saver.py

```diff
@@ -0,0 +1,44 @@
+from pathlib import Path
+import json
+
+from card import Card
+
+from utils import is_parent
+
+
+class Saver:
+    def save(self, card: Card, dir_path='.'):
+        """Save a given card to a given path."""
+        card_path = self.get_card_path(card, dir_path)
+        metadata_path = card_path / "metadata.json"
+        image_path = card_path / "image.jpg"
+
+        metadata = {
+            "name": card.name,
+            "creator": card.creator,
+            "riddle": card.riddle,
+            "solution": card.solution,
+            "image_path": str(image_path)
+        }
+        with metadata_path.open('w') as f:
+            json.dump(metadata, f)
+
+        card.image.image.save(image_path)
```

Comment:
I added a method `CryptImage.save_image` https://github.com/roy-cohen273/cardazim/pull/4/commits/53a7bdecfc8f46853fbbdf5ebb70ebe112c0bd6b

I didn't want to have this method in `Card` because that can be interpreted as saving the entire `CryptImage`.